apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/OneOfInputObjectTemplate.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import GraphQLCompiler
+import TemplateString
+
+struct OneOfInputObjectTemplate: TemplateRenderer {
+    
+  let graphqlInputObject: GraphQLInputObjectType
+  
+  let config: ApolloCodegen.ConfigurationContext
+  
+  let target: TemplateTarget = .schemaFile(type: .inputObject)
+  
+  func renderBodyTemplate(
+    nonFatalErrorRecorder: ApolloCodegen.NonFatalError.Recorder
+  ) -> TemplateString {
+    let (validFields, deprecatedFields) = graphqlInputObject.fields.filterFields()
```

Comment:
Getting warnings here for these properties not being used. Can we remove this line?